Wiki-style-guide.md

```diff
@@ -0,0 +1,13 @@
+## Text markup
+
+* Use markdown headers since we can create links to headers in the wiki. This is not possible with long lists or "headers" that are just bold.
+* Use quotes (`""`) for single-line quotes. This includes button names.
+* Use markdown block quotes for large quotations.
```

Comment:
Maybe give examples inline for what the code/display for these looks like (markdown headers, markdown block quotes, etc.).

If possible, try to also explain/show what not to do.

Reply:
done